components/brave_today/browser/brave_news_controller.cc

```diff
@@ -45,6 +48,11 @@
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
 namespace brave_news {
+namespace {
+// The favicon size we desire. The favicons are rendered at 24x24 pixels but
+// they look quite a bit nicer if we get a 48x48 pixel icon and downscale it.
+constexpr uint32_t kDesiredFaviconSizePixels = 48;
```

Comment:
does the favicon service actually store these higher resolution images? Is that something new?

Reply:
It definitely seems to, at least for some icons. They look significantly better when I get the 48x48px icons.